docs/source/daml/reference/templates.rst

```diff
@@ -71,7 +71,7 @@ Signatory parties
   ``NameOfTemplate requires authorizers Party1,Party2,Party, but only Party1 were given.``
 - When a signatory consents to the contract creation, this means they also authorize the consequences of :ref:`choices <daml-ref-choices>` that can be exercised on this contract.
 - The contract instance is visible to all signatories (as well as the other stakeholders of the contract). That is, the compiler automatically adds signatories as observers.
-- You **must** have least one signatory per template. You can have many, either as a comma-separated list or reusing the keyword. You could pass in a list (of type ``[Party]``).
+- Each template **must** have least one signatory declaration. This consists of the `signatory` keyword followed by a single name, a comma-separated list of names, or a list of type `[Party]`.
```

Comment:
I don’t think this is quite right. You really need to have one signatory not just one signatory declaration. E.g., `signatory []` will be an error since you have an empty list of signatories.

Reply:
To me, it is also not clear what a "name" should mean here. What you need to specify after signatory is a comma-separated list of expressions of type `exists a. IsParties a => a`. Instances of `IsParties` are `Party`, `[Party]`, `Optional Party` and `Set Party`. However, that's probably a bit too specific for this document.

Reply:
Excellent points; addressed both in the second commit.